core/src/main/scala/com/dimafeng/testcontainers/FixedHostPortGenericContainer.scala

```diff
@@ -10,6 +11,7 @@ class FixedHostPortGenericContainer(imageName: String,
                                     command: Seq[String] = Seq(),
                                     classpathResourceMapping: Seq[FileSystemBind] = Seq(),
                                     waitStrategy: Option[WaitStrategy] = None,
+                                    startupCheckStrategy: Option[StartupCheckStrategy] = None,
```

Comment:
Would you mind moving this argument to the end to make it backward compatible for the positional use?